charts/tfy-grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: tfy-grafana
 description: Truefoundry grafana provisioner
-version: 0.1.6
+version: 0.1.7
```

Comment:
Is the dashboard final? Maybe uae rc versions till then

Reply:
good point. done.